### PR TITLE
Add the Binding.scala·TodoMVC example

### PR DESCRIPTION
**Pull request by Atry**

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -48,7 +48,9 @@ Use these Open Source projects to see how others have used Scala.js to build the
 - [Play and Scala.js Showcase](https://github.com/hussachai/play-scalajs-showcase), by Hussachai Puripunpinyo
 - [Papa-Carlo Incremental Parser Demo](http://lakhin.com/projects/papa-carlo/demo/), by Eliah-Lakhin
 - [Ray-Tracer](http://lihaoyi.github.io/workbench-example-app/raytracer.html), a ray-tracer written in Scala.js by Li Haoyi
-- [TodoMVC](http://lihaoyi.github.io/workbench-example-app/todo.html), an implementation of the [TodoMVC example application](http://todomvc.com/) using Scala.js, Scalatags, Scala.Rx and scala-js-dom, by Li Haoyi
+- [TodoMVC example applications](http://todomvc.com/)
+  - [Scala.Rx·TodoMVC](http://lihaoyi.github.io/workbench-example-app/todo.html), an implementation of the using Scala.js, Scalatags, Scala.Rx and scala-js-dom, by Li Haoyi
+  - [Binding.scala·TodoMVC](http://todomvc.com/examples/binding-scala/), an implementation of the using Scala.js, Binding.scala and upickle, by Yang Bo
```
Comment (sjrd) [inline community/index.md]: 1. "an implementation **of the using**" (there's something missing there), in both bullets
2. There's a weird character `·` before "TodoMVC" in the two links. The same also appears in the commit message. It should be a normal space, IMO.

**Comment by Atry** [comment]
@sjrd I have fixed the issues you point out. Would you like to review it again?

**Comment by sjrd** [comment]
OK could you squash the commits, keeping only the message of the first one?

**Comment by Atry** [comment]
See #311

**Comment by sjrd** [comment]
You didn't have to create a new PR. Instead, you can force-push on the branch of the existing PR.

**Pull request closed without merging**